Gopkg.lock

```diff
@@ -16,6 +16,17 @@
   revision = "050b16d2314d5fc3d4c9a51e4cd5c7468e77f162"
   version = "v0.17.0"
 
+[[projects]]
+  digest = "1:fd5f5c00789da5a6ae78b6c6050eccea46bcd4a519a4913ac82d22bafd7fd368"
+  name = "github.com/Azure/azure-sdk-for-go"
```

Comment:
Should `github.com/Azure/azure-sdk-for-go` be added as a top level `constraint` in Gopkg.toml`?

Reply:
I don't think that's necessary, because I'm using a versioned import path for azure sdk. See [here](https://github.com/GoogleContainerTools/skaffold/blob/88f8e32642a289beb04ebfdf61ac6b423f258c94/pkg/skaffold/build/acr/acr_build.go#L27)